src/Entity/YesplanEvent.php

```diff
@@ -97,10 +97,10 @@ class YesplanEvent
     #[ORM\Column(nullable: true)]
     private ?bool $inPresaleDateUpdated = null;
 
-    #[ORM\Column]
+    #[ORM\Column(nullable: true)]
```

Comment:
Shouldn't this (`nullable: true`) be reflected in an updated migration?

Reply:
Nope, it was already this way in previous migrations, but when adding the entities `nullable: true` was forgotten.